templates/jdk-common/resources/home/liferay/.bashrc

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
+
```

Comment:
@Wykenz, was this missing and then you added it back?

Reply:
shellcheck was showing an error in the first line because this was missing